apps/sms/views/inbox/js/inbox.js

```diff
@@ -585,29 +568,29 @@ var InboxView = {
     // Request and render all threads with drafts
     // or thread-less drafts.
     return Drafts.request(force).then(() => {
-      Drafts.forEach(function(draft, threadId) {
-        var thread;
-        if (threadId) {
+      for (var draft of Drafts.getAll()) {
+        var node = document.getElementById(
+          `thread-${draft.threadId || draft.id}`
+        );
+
+        if (draft.threadId) {
           // Find draft-containing threads that have already been rendered
           // and update them so they mark themselves appropriately
-          var el = document.getElementById('thread-' + threadId);
-          if (el) {
-            thread = Threads.get(threadId);
-            this.updateThread(thread);
+          if (node) {
```

Comment:
same remark than below: using Threads.get can we be sure than the thread was rendered? Maybe not... but maybe we can keep this information in an array? Or maybe it's not worth it? What do you think?